src/invoice.rs

```diff
@@ -120,9 +140,15 @@ impl std::fmt::Display for RgbInvoice {
         let amt = self.owned_state.to_string();
         write!(f, "{}", self.transport)?;
         if let Some(contract) = self.contract {
-            write!(f, "{}", contract.to_baid58())?;
+            write!(f, "{}/", contract.to_baid58())?;
+        } else {
+            write!(f, "{ANY}/")?;
+        }
+        if let Some(iface) = self.iface.clone() {
+            write!(f, "{}/", iface)?;
```

Comment:
```suggestion
            write!(f, "{iface}/")?;
```

Otherwise rust linter will be failing CI

Reply:
cargo check/clippy locally did not report this and CI also passed

not sure why this was not detected, good catch. updated